### PR TITLE
improve url construction

### PR DESCRIPTION
**Pull request by muhammad-ammar**
https://openedx.atlassian.net/browse/EDUCATOR-1822

### CODE REVIEW

**Comment by muhammad-ammar** [comment]
@mushtaqak @Qubad786 This is ready for review.

**Comment by Qubad786** [comment]
@muhammad-ammar does this also include fix to `scrub_url`?

File: control/tests/test_deliver_cielo.py [hunk 1]
```diff
@@ -146,13 +146,13 @@ def test_transcript_flow(self):
                     v=CIELO24_API_VERSION,
                     job_id='000-111-222',
                     target_language='TARGET_LANG',
-                    callback_url=build_url(
+                    callback_url='{}?job_id={}&iwp_name={}&lang_code={}&org={}&video_id={}'.format(
```
Comment (mushtaqak) [inline control/tests/test_deliver_cielo.py]: I think we should use format variables here like `{cielo_callback_base_url}?job_id={job_id}&iwp_name={iwp_name}&lang_code={lang_code}&org={org}&video_id={video_id}`

File: control/veda_deliver_cielo.py [hunk 2]
```diff
@@ -117,13 +117,13 @@ def perform_transcript(self, job_id, lang_code):
         """
         Request cielo24 to generate transcripts for a video.
         """
-        callback_url = build_url(
+        callback_url = '{}?job_id={}&iwp_name={}&lang_code={}&org={}&video_id={}'.format(
```
Comment (mushtaqak) [inline control/veda_deliver_cielo.py]: same as above

File: control/veda_deliver_cielo.py [hunk 3]
```diff
@@ -136,7 +136,7 @@ def perform_transcript(self, job_id, lang_code):
             api_token=self.api_key,
             priority=self.turnaround,
             transcription_fidelity=self.fidelity,
-            options=json.dumps({"return_iwp": ["FINAL"]})
+            options=json.dumps({"return_iwp":["FINAL"]})
```
Comment (mushtaqak) [inline control/veda_deliver_cielo.py]: wasn't space better for quality ?

File: control/tests/test_deliver_cielo.py (hunk 1 above)
Comment (muhammad-ammar) [reply control/tests/test_deliver_cielo.py]: This will make the line longer and pylint will complain.

File: control/veda_deliver_cielo.py (hunk 2 above)
Comment (muhammad-ammar) [reply control/veda_deliver_cielo.py]: This will make the line longer and pylint will complain.

File: control/veda_deliver_cielo.py (hunk 3 above)
Comment (muhammad-ammar) [reply control/veda_deliver_cielo.py]: it was causing issues during url encoding so we removed it

**Pull request merged**